pyfar/dsp/fft.py

```diff
@@ -301,16 +404,138 @@ def _n_bins(n_samples):
     ---------
     n_samples : int
         Number of samples
+    complex : bool
+        Flag which indicates if the time data are real or complex-valued.
 
     Returns
     -------
     n_bins : int
         Resulting number of frequency bins
 
     """
-    if _is_odd(n_samples):
-        n_bins = (n_samples+1)/2
+    if complex:
+        n_bins = n_samples
     else:
-        n_bins = n_samples/2+1
+        n_bins = n_samples // 2 + 1
 
     return int(n_bins)
+
+
+def _calc_n_samples_from_frequency_data(num_freq_bins, complex=False):
+    """
+    Helper function to calculate the number of samples resulting from
+    an inverse FFT of a spectrum with n_freq_bins
+
+    Paramters
+    ---------
```

Comment:
```suggestion
    Parameters
    ----------
```
See also below, this typo breaks the correct generation of the Sphinx docs.
There's also a couple of other typos. I'd recommend installing a spell checker plug-in such as cSpell :)